src/server/routers/account.ts

```diff
@@ -10,7 +11,24 @@ export const accountRouter = t.router({
         },
       },
     });
-
+    return accounts
+  }),
+  chooseProvider: t.procedure.use(isAuthed)
```

Comment:
if i m not wrong , the function `me` is only used to get github or gitlab account .
so no need to create a new function and just change the `me` function and that's it